packages/cli/e2e/__tests__/destroy.spec.ts

```diff
@@ -30,7 +30,6 @@ describe('destroy', () => {
       directory: path.join(__dirname, 'fixtures', 'deploy-project'),
       env: { PROJECT_LOGICAL_ID: projectLogicalId },
     })
-    expect(result.status).not.toBe(0)
```

Comment:
Does the test fail with this assertion? Normally `npx checkly destroy` won't get the confirmation prompt and the command should timeout. In that case, the exit code should be `null` and the test should pass.

Reply:
It would be nice to replace this with some other assertion that the timeout is reached. I think that that would help make sure `checkly destroy` keeps working as expected in the future.

Reply:
I looked and I think it's because prompts is capturing the interrupts and calling this `onCancel` callback https://github.com/terkelg/prompts/issues/27